lua/mason-registry/eslint_d/init.lua

```diff
@@ -0,0 +1,11 @@
+local Pkg = require "mason-core.package"
+local npm = require "mason-core.managers.npm"
+
+return Pkg.new {
+    name = "eslint_d",
+    desc = [[Eslint_d is a very fast eslint server for JavaScript and TypeScript]],
```

Comment:
```suggestion
    desc = [[Makes eslint the fastest linter on the planet]],
```

(I tend to source these descriptions directly from upstream)

Reply:
ah got it i'll remember that for next time :D